### PR TITLE
Selectionsort

### LINKED ISSUE #2154: Selection Sort
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [ ] Dart - @pooja-git11
- [x] Go
- [x] Java
- [x] JS
- [x] Kotlin
- [x] Php - @BhardwajShweta25
- [x] Python
- [x] Readme
- [x] Ruby
- [x] Rust - @raksha009
- [ ] TypeScript - @lakshyajit165

### PR DESCRIPTION
**Pull request by lakshyajit165**
Fixes #2154 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Selection Sort logic in Typescript

#### Languages Used:
- Typescript 

#### Files Added:
- Selection_Sort.ts

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by lakshyajit165** [comment]
@jainaman224 and @asha15. Please review my PR.

**Comment by lakshyajit165** [comment]
@jainaman224 @asha15 @SakshiShreya  - A gentle follow up on this.

File: Selection_Sort/Selection_Sort.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            numbers[i] = numbers[min];
+            numbers[min] = temp;
+        }
+    }
+
+    return numbers;
+}
+
+// array of numbers (INPUT)
+const numbers: number[] = [10, 9, 8, 1, 5, 2, 4, 7, 3];
```
Comment (raghav-dalmia) [inline Selection_Sort/Selection_Sort.ts]: Take input dynamically

File: Selection_Sort/Selection_Sort.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+// OUTPUT ([ 1, 2, 3, 4, 5, 7, 8, 9, 10 ])
+console.log(Selection_Sort(numbers));
+
```
Comment (raghav-dalmia) [inline Selection_Sort/Selection_Sort.ts]: sample input/output should be here

File: Selection_Sort/Selection_Sort.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// function for selection sort logic
```
Comment (raghav-dalmia) [inline Selection_Sort/Selection_Sort.ts]: add comments in the code

**Comment by raghav-dalmia** [comment]
Also, squash your commits.

File: Selection_Sort/Selection_Sort.ts (hunk 2 above)
Comment (lakshyajit165) [reply Selection_Sort/Selection_Sort.ts]: @raghav-dalmia I have mentioned the sample input/output already. Do you want me to add more examples?

File: Selection_Sort/Selection_Sort.ts (hunk 1 above)
Comment (lakshyajit165) [reply Selection_Sort/Selection_Sort.ts]: @raghav-dalmia I had discussed this with @jainaman224, he asked to take input this way, could you suggest how else to take input in typescript? (I had initially thought of using alerts in a html page)

File: Selection_Sort/Selection_Sort.ts (hunk 2 above)
Comment (raghav-dalmia) [reply Selection_Sort/Selection_Sort.ts]: I am saying, it should be at the bottom of the file.

Comment (lakshyajit165) [reply Selection_Sort/Selection_Sort.ts]: @raghav-dalmia I got it. Will do it in a while and let you know.

**Comment by lakshyajit165** [comment]
@raghav-dalmia. I have updated the changes. Please review.

**Pull request closed without merging**